### PR TITLE
docs: adjust sample code to changes in resolver requirements

### LINKED ISSUE #196: Sample code not working
This code is not working due to the breaking changes in 'ethr-did-resolver' library. Please provide the working sample for creating and verifying disclosure request and credentials.

### PR DESCRIPTION
**Pull request by mirceanis**
fixes #196

### CODE REVIEW

File: docs/guides/index.md [hunk 1]
```diff
@@ -13,32 +13,31 @@ In your application, you must first configure your uPort object with an identifi
...
- 
+import { Resolver } from 'did-resolver'
+import { getResolver } from 'ethr-did-resolver'
+
+const providerConfig = { rpcUrl: 'https://mainnet.infura.io/<YOUR INFURA PROJECT ID>' }
+const resolver = new Resolver(getResolver(providerConfig))
+
 // For ethereum based addresses (ethr-did)
 const credentials = new Credentials({
   appName: 'App Name',
```
Comment (justinmchase) [inline docs/guides/index.md:24]: I think that the `appName` and `privateKey` fields are no longer available and should be replaced with `signer: SimpleSigner(privateKey)`

File: docs/guides/index.md [hunk 2]
```diff
@@ -13,32 +13,31 @@ In your application, you must first configure your uPort object with an identifi
...
- 
-Finally, we continue to support older uPort identities described by an [MNID](http://github.com/uport-project/mnid)-encoded Ethereum address. These identifiers can be expressed as a DID via the 'uport' DID method: `did:uport:<mnid>`
-```javascript
-// For legacy application identity created on App Manager
 const credentials = new Credentials({
   appName: 'App Name',
-  did: 'did:uport:2nQtiQG...',  //append MNID encoded address using did:uport method 
-  privateKey: process.env.PRIVATE_KEY
+  did,
+  privateKey,
```
Comment (justinmchase) [inline docs/guides/index.md:39]: I think you need to set `signer: SimpleSigner(privateKey)` here too

File: docs/guides/tutorial.md [hunk 3]
```diff
@@ -52,9 +52,15 @@ $ node
...
 ```js
+import { Resolver } from 'did-resolver'
+import { getResolver } from 'ethr-did-resolver'
+const providerConfig = { rpcUrl: 'https://mainnet.infura.io/<YOUR INFURA PROJECT ID>' }
+const resolver = new Resolver(getResolver(providerConfig))
+
 var credentials = new uport.Credentials({
   did: 'did:ethr:0xbc3ae59bc76f894822622cdef7a2018dbe353840',
-  privateKey: '74894f8853f90e6e3d6dfdd343eb0eb70cca06e552ed8af80adadcc573b35da3'
+  privateKey: '74894f8853f90e6e3d6dfdd343eb0eb70cca06e552ed8af80adadcc573b35da3',
```
Comment (justinmchase) [inline docs/guides/tutorial.md:62]: `signer: SimpleSigner(...)` here also. Are you sure you want an actual hard coded private key here in an example?

File: docs/guides/index.md (hunk 2 above)
Comment (mirceanis) [reply docs/guides/index.md:39]: It is true that the direct private key parameter could be deprecated in favor of the `Signer` abstraction but it is not the goal of this PR.

File: docs/guides/index.md (hunk 1 above)
Comment (mirceanis) [reply docs/guides/index.md:24]: The options are still available.
Also, at this point there's not much difference between sending the private key directly or using the `SimpleSigner` which also needs direct access to the key.

File: docs/guides/tutorial.md (hunk 3 above)
Comment (mirceanis) [reply docs/guides/tutorial.md:62]: This private key has been compromised the moment it touched git.
A few lines above there are instructions on how to store the key.

**Pull request merged**